chapter-template/release-notes-template.md

```diff
@@ -19,6 +19,19 @@ The release_year-release_month release bundle for SystemLink Enterprise has been
 - Chart Name and version
     - Description of breaking change.
 
+## Upgrade Considerations
+
+### RabbitMQ Version
+
+SystemLink Enterprise includes a deployment of the [RabbitMQ](https://www.rabbitmq.com/) message bus. It is not possible to skip minor versions when updating RabbitMQ. This may prevent direct upgrades between versions of the SystemLink Enterprise product. The table below shows the version of the RabbitMQ dependency for each released version of SystemLink Enterprise.
+
+| RabbitMQ Version | First SystemLink Enterprise Version | Last SystemLink Enterprise Version |
```

Comment:
@MelissaHilliard I like your alternate suggestion for a table that shows the required upgrade path. However, since we are only managing a single version bump at the time, that table wouldn't be particularly interesting, and we're probably 6 months to a year from needing to update again. I think we should re-think this table layout at that time.

Reply:
I can get behind that